### PR TITLE
New bot request - Cloudtrail encrypt log files with kms

### PR DESCRIPTION
**Pull request by Roiabr**

### CODE REVIEW

**Comment by RavidAtias** [comment]
I see this one very similar to https://github.com/dome9/cloud-bots/pull/52/files
Maybe use utils for shared logic?

File: bots/cloudtrail_encrypt_log_files_with_kms.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    kms_client = boto_session.client('kms')
+
+    cloudTrail_name = entity['id']
+    account_id = entity['accountNumber']
+
+    try:
+        if not params:
+            key_id = create_customer_key(kms_client, iam_client, account_id)
+        else:
+            key_id = params[0]
```
Comment (RavidAtias) [inline bots/cloudtrail_encrypt_log_files_with_kms.py:87]: If you added option for param please add to documentation

File: bots/cloudtrail_encrypt_log_files_with_kms.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+        # Enable the encryption in the cloudtrail
+        cloudtrail_client.update_trail(
```
Comment (RavidAtias) [inline bots/cloudtrail_encrypt_log_files_with_kms.py:90]: What will happen if there already a key_id like this?

**Comment by Roiabr** [comment]
At Guy's suggestion, I split the bot into 2 bots, one of which would create a new key.
And another bot that the user will send the key as a parameter (I will write the bot later)
That way we will not interfere with the user cloud environment unless he wants to.

**Pull request merged**